pkg/osquery/runtime/runner.go

```diff
@@ -290,6 +291,13 @@ func (r *Runner) launchOsqueryInstance() error {
 		"args", strings.Join(o.cmd.Args, " "),
 	)
 
+	// remove osquery socket if exists
+	if err := backoff.WaitFor(func() error {
```

Comment:
Do you think this needs the retry?

Reply:
Actually I added the retry because I was thinking of windows and that you can't delete an opened file ... but on windows we create a random socket path so it doesn't matter. I'll remove it